src/main/java/dinkplugin/util/WorldUtils.java

```diff
@@ -105,6 +106,12 @@ public boolean isPlayerOwnedHouse(int regionId) {
     public boolean isSafeArea(Client client) {
         int regionId = getLocation(client).getRegionID();
 
+        if (isAmascutTombs(regionId)) {
+            // ToA is technically a dangerous activity, but multiple attempts can be permitted
+            // the real TOA death is detected via game message in death notifier
+            return true;
+        }
```

Comment:
Currently this makes ToA safe for HCIM, I have a feeling that's not how it works

Reply:
> https://youtu.be/GzwCCghNckw?t=36 had the death message, not sure what invocations were used so this should be ok

FWIW, this looks like Hardcore Run invocation where you get only 1 team life, however he was also alone in the raid, so his 1 death was considered a wipe (and full team wipes result in a "raid life" loss), and also 1 wipe (hardcore run) is considered a failed raid, so you're thrown out.